tests/metrics/test_regression.py

```diff
@@ -35,3 +35,18 @@ def test_ok(metric_pairs, compute):
         assert isinstance(result, da.Array)
     expected = m2(a, b)
     assert abs(result - expected) < 1e-5
+
+
+@pytest.mark.parametrize("squared", [True, False])
+def test_mse_squared(squared):
+    m1, m2 = (
+        getattr(dask_ml.metrics, "mean_squared_error"),
+        getattr(sklearn.metrics, "mean_squared_error"),
+    )
```

Comment:
No need for the getattr, since we aren't parametrizing over metrics.

```suggestion
    m1 = dask_ml.metrics.mean_squared_error
    m2 = sklearn.metrics.mean_squared_error
```